helix-core/src/comment.rs

```diff
@@ -94,12 +94,56 @@ pub fn toggle_line_comments(doc: &Rope, selection: &Selection, token: Option<&st
     Transaction::change(doc, changes.into_iter())
 }
 
+/// Return the comment token of the current line if it is commented.
+///
+/// Return None otherwise.
+pub fn get_comment_token<'a>(line: &RopeSlice, tokens: &'a [String]) -> Option<&'a str> {
+    // TODO: don't continue shebangs
```

Comment:
This TODO was carried over from #1937, but I'm not sure what it means.

Reply:
About shebangs in general: https://en.wikipedia.org/wiki/Shebang_(Unix)

We shouldn't continue shebangs as if they were regular comments since they only take up one line